### ISSUE TITLE
Incomplete MLB Roster

### ISSUE THREAD

**Issue by technicalogical** [mattheworres/phpdraft#11]
A few picks in I noticed a few player missing.  Votto and Rizzo were both off of the list.  I compiled a list from Steamer with a fairly complete list.  If you would like to use it you can snag it from: https://drive.google.com/file/d/0B5QKlck1wmIYcGtlWVFMUnV0bDA/view?usp=sharing

It's kinda messed up though...  All players without a team were set to DET and some of the players names got messed up when doing find/replace on the team names.

**Comment by mattheworres** [comment]
@technicalogical ugh! Well I'll accept PRs against the file, and I guess will direct folks to use the CSV from the master branch to fix any issues: https://github.com/mattheworres/phpdraft/blob/master/resources/ProPlayers_MLB_2016.csv

It's odd that guys like Votto and Rizzo were missing. Dangit!

I use CBS Sports' API for the sports data input, but generally they've been pretty good.

I'll look into maybe starting fresh with your Steamer list instead -- and maybe move to using it in the future.

**Comment by mattheworres** [comment]
Looking through the data Steamer provides, and it seems to be more complete, but it may also be a little less scrubbed. I noticed a fair amount of duplicates (either one with a team, and one without one) and a decent number of players without teams (and those two issues also seem fairly intertwined).

I'll have to take a look at this on the Windows side of my machine (Excel for Mac is single-threaded so a list this large with a COUNTIFS statement gets pretty sluggish), but I'll try and work through their list and scrub the data a little more to see if I can come up with a more complete CSV.

**Comment by mattheworres** [comment]
@technicalogical I ended up using the same source I did originally, but went a different way. For some reason the different ways CBS Sports sorts their data results in some players being left out in odd ways. Regardless, looks like I got it right this time. Votto and Rizzo are available, and apparently I was missing players like Josh Harrison (3B, PIT) too. Doesn't really make much sense why, but it's fixed on the master branch: https://github.com/mattheworres/phpdraft/blob/master/resources/ProPlayers_MLB_2016.csv
